tests/database/test_journal_db.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from evm.db.backends.memory import MemoryDB
+from evm.db.journal import JournalDB
+
+
+@pytest.fixture
+def memory_db():
```

Comment:
ISTM this fixture is not used anywhere other than the journal_db below, so maybe inline it there?